product_order_noname/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    'name': 'Product Order No Name',
+    "version": "12.0.1.0.0",
+    'summary': 'Speedup product retrieve',
+    'category': 'Others',
+    "price": 30,
```

Comment:
remove price

Reply:
Use the following file as a template:
 - https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py